### ISSUE TITLE
Does not work in Foundry VTT 0.7.3+

### ISSUE THREAD

**Issue by sgrigson** [luvolondon/fvtt-module-jitsiwebrtc#31]
On load, returns 

```
Uncaught ReferenceError: WebRTCInterface is not defined
    at jitsirtc.js:8
```

After that it throws CORS errors about the /easyrtc endpoint, I suspect because jitsirtc.js never loaded properly because of the `WebRTCInterface` not being defined.

Note, my existing configuration works without issue on 0.6.6.

**Comment by bekriebel** [comment]
This is a known issue. There is a development branch in progress with fixes, but it is not yet complete: https://github.com/bekriebel/fvtt-module-jitsiwebrtc/tree/rtc-next

This will provide at least basic functionality, but still has some known bugs around switching devices and possible other issues.

**Comment by bekriebel** [comment]
Fixed in version 0.4.0.
